### DISCUSSION TITLE
I get an error when logging out in keycloak v22.0.5.

### DISCUSSION THREAD

**Discussion by Phanhieu460** [keycloak/keycloak#24451] [Archive]
![image](https://github.com/keycloak/keycloak/assets/75322785/28a6f0c0-cbef-4cd9-bd67-a2fe2e5f538f)
I get error when logout i use keycloak-angular v6.1.0, keycloak-js v11.0.3 and keycloak 22.0.5

**Comment by jonkoops** [comment]
You should use the same version of Keycloak JS as the Keycloak version you are running on your server. This might help solve your issue. If not, then you should share the code you are using to log out so that the issue can be reproduced.

**Reply by jsorah** [reply]
This looks very much like an older client using the legacy `redirect_uri` parameter and needs to be migrated to `post_logout_redirect_uri` + `id_token_hint` as mentioned in KC 18 upgrade notes -> https://www.keycloak.org/docs/latest/upgrading/index.html#openid-connect-logout

Which should be the case in KeycloakJS versions 18+.
